addons/mail/static/src/core/common/message_model.js

```diff
@@ -121,16 +121,7 @@ export class Message extends Record {
     }
 
     get datetime() {
-        if (!this._datetime) {
-            this._datetime = toRaw(this.date ? deserializeDateTime(this.date) : this.now);
```

Comment:
`toRaw` is not used in new code?

Reply:
I don't know why we needed this toRaw to begin with.